beetsplug/audible.py

```diff
@@ -621,4 +621,13 @@ def get_item_region(item):
         result = get_audible_album_region(album_url)
     else:
         result = None
-    return result
\ No newline at end of file
+    return result
+
+
+# UI color names fallbacks.
+UI_COLOR_NAMES = {
```

Comment:
I chose this solution. With beets==2.0.0, the UI colors are configurable. With beets<=1.6.0, the colors are hardcoded.